### PR TITLE
Creating setuphandler to allow in-game set up to be queried via API

### PR DESCRIPTION
**Pull request by pwdel**
See ticket: https://github.com/orgs/openpredictionmarkets/projects/4/views/1?pane=issue&itemId=76257552

To test, load branch and visit:

http://localhost/api/v0/setup

### CODE REVIEW

File: backend/handlers/setup/setuphandler.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		http.Error(w, "Failed to load economic config", http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	err = json.NewEncoder(w).Encode(appConfig.Economics)
+	if err != nil {
+		http.Error(w, "Failed to encode response", http.StatusInternalServerError)
+	}
+}
```
Comment (ajlacey) [inline backend/handlers/setup/setuphandler.go:22]: Future work question: should we also think about letting an admin user update the setup via a POST to this same endpoint?

File: backend/handlers/setup/setuphandler_test.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		MockConfigLoader func() (*setup.EconomicConfig, error)
+		ExpectedStatus   int
+		ExpectedResponse string
+		IsJSONResponse   bool
+	}{
+		{
+			Name: "successful load",
+			MockConfigLoader: func() (*setup.EconomicConfig, error) {
+				return setup.LoadEconomicsConfig()
+			},
```
Comment (ajlacey) [inline backend/handlers/setup/setuphandler_test.go:26]: I like the way you did this! Can we add an error case as well, where the config fails to load?

**Review by ajlacey — APPROVED** [review]
LGTM!

It would be useful to add in the error cases when testing GetSetupHandler, but this looks good!

File: backend/handlers/setup/setuphandler.go (hunk 1 above)
Comment (pwdel) [reply backend/handlers/setup/setuphandler.go:22]: Great idea!

Comment (pwdel) [reply backend/handlers/setup/setuphandler.go:22]: Put together a draft ticket on this for project v0.0.5 https://github.com/orgs/openpredictionmarkets/projects/5?pane=issue&itemId=76289189

File: backend/handlers/setup/setuphandler_test.go (hunk 2 above)
Comment (pwdel) [reply backend/handlers/setup/setuphandler_test.go:26]: Added and tagged a ticket within refactoring for this.

https://github.com/orgs/openpredictionmarkets/projects/6/views/1?pane=issue&itemId=76289411

**Pull request merged**